apps/mvp/src/components/ConnectWalletsFlow.vue

```diff
@@ -40,7 +40,7 @@ const { getEthersLedgerAddresses } = useLedger()
 const { getEthersTrezorAddresses } = useTrezor()
 const { user } = useUser()
 const { detectActiveNetwork, switchEthersNetwork } = useWallets()
-const { connectWalletConnectV2 } = useWalletConnect()
+const { connectWalletConnectV2, walletConnectSelectedAccount } = useWalletConnect()
```

Comment:
Generally curious, would it ever make sense to do something like:

```ts
const walletConnect = useWalletConnect()
...
watch(walletConnect.selectedAccount, () => {
...
}
```

`composableName.someVariableOrMethodName` might be nice for organization (and less redundancy).

Not suggesting this for the PR.

Reply:
cool pattern.